apps/ledger-live-desktop/src/renderer/screens/platform/v2/Catalog/Browse/Search.tsx

```diff
@@ -22,6 +22,8 @@ export function Search({ categories, search }: Props) {
     [categories.categories, t],
   );
 
+  const selected = options.find(opt => opt.value === categories.selected);
```

Comment:
This should be done in a memo to avoid recomputing on every render, also it should be better to also move the fallback `options[0]` in the memo directly
We can also rename the const to `selectedOption` or `defaultOption`